code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -0,0 +1,109 @@
+/obj/structure/stool/bed/chair/vehicle/wheelchair
+	name = "wheelchair"
+	nick = "cripplin' ride"
+	desc = "A chair with fitted wheels. Used by handicapped to make life easier, however it still requires hands to drive."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "wheelchair"
+	anchored = 0
+
+	speed = 3
+
+	health = 50
+	max_health = 50
+
+	var/image/wheel_overlay
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/New()
+	.=..()
+	wheel_overlay = image("icons/obj/objects.dmi", "[icon_state]_overlay", MOB_LAYER + 0.1)
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/buckle_mob()
+	.=..()
+	update_icon()
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/unbuckle()
+	.=..()
+	update_icon()
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/update_icon()
+	..()
+	if(buckled_mob)
+		overlays |= wheel_overlay
+	else
+		overlays -= wheel_overlay
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/can_buckle(mob/M, mob/user)
+	if(M != user || !ishuman(user) || !Adjacent(user) || user.restrained() || user.stat || user.buckled || destroyed) //Same as vehicle/can_buckle, minus check for user.lying
+		return 0
+	return 1
+
+/obj/structure/stool/bed/chair/vehicle/wheelchair/proc/check_hands(var/mob/user)
+	//Returns a number from 0 to 4 depending on usability of user's hands
+	//User with no hands gets 0
+	//User with one hand holding something gets 1
+	//User with one empty hand gets 2
+	//User with two hands both holding something gets 2
+	//User with one empty and one full hand gets 3
+	//User with two empty hands gets 4
+
+	//Wheelchair's speed depends on the resulting value
+	var/mob/living/carbon/human/H = user
```

Comment:
Why do you assume the user is human?
Indent the organ checks under an istype(H)